recipes/pestpp/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+mkdir build
+cd build
```

Comment:
`pushd` is preferable, and `popd` when done.

Reply:
I updated the scripts